x-pack/plugins/synthetics/common/constants/monitor_defaults.ts

```diff
@@ -158,16 +161,22 @@ export const DEFAULT_FIELDS: MonitorDefaults = {
     ...DEFAULT_HTTP_SIMPLE_FIELDS,
     ...DEFAULT_HTTP_ADVANCED_FIELDS,
     ...DEFAULT_TLS_FIELDS,
+    [ConfigKey.FORM_MONITOR_TYPE]: FormMonitorType.HTTP,
```

Comment:
This new abstraction is needed because browser monitors are now split into two types, multistep and single. Since this is now part of the monitor type, it needs to be defined on defaults. Likewise, this also ensures that new monitors created from Uptime are created with the appropriate spec to be able to be compatible with Synthetics edit page.